package.json

```diff
@@ -30,7 +30,14 @@
   },
   "homepage": "http://material-ui.com/",
   "dependencies": {
-    "react-draggable2": "^0.5.1"
+    "react": ">=0.14.0-rc1",
+    "react-dom": ">=0.14.0-rc1",
```

Comment:
@oliviertassinari You do if you wish to run the build task in the root of the repository without dependency warnings. Newer versions of npm no longer auto-install missing peerDependencies, instead they throw an error.